radicle-cli/src/commands/node.rs

```diff
@@ -20,17 +21,19 @@ pub const HELP: Help = Help {
     usage: r#"
 Usage
 
-    rad node status
-    rad node (start|stop)
-    rad node connect <nid> <addr>
-    rad node routing
-    rad node tracking [--repos|--nodes]
+    rad node status [<option>...]
+    rad node start [<option>...]
+    rad node stop [<option>...]
+    rad node connect <nid> <addr> [<option>...]
+    rad node routing [<option>...]
+    rad node tracking [--repos|--nodes] [<option>...]
```

Comment:
Tbh I would remove the `[<option>...]` you added to all commands, it's just noise.

Reply:
correct, but then they wouldn't be uniform across all commands and will diverge over time, it's a small cost to pay.